From ca658f1bdfd2ad40b3f9cdaf9491dda7f780b541 Mon Sep 17 00:00:00 2001
From: Madhusudhan S <madhusudhan.s@intel.com>
Date: Wed, 14 Nov 2018 16:28:15 +0530
Subject: [PATCH] [CELADON]Enabling display timeout on IVI.

Changing SCREEN_DIM_WAKE_LOCK to PARTIAL_WAKE_LOCK
since SCREEN_DIM_WAKE_LOCK never allows to timeout
display.

Change-Id: Ia8312fcefe70902f185e4ad7423073bc1d529ad9
Tracked-on: OAM-69117
Signed-off-by: Madhusudhan S <madhusudhan.s@intel.com>
---
 service/src/com/android/car/systeminterface/WakeLockInterface.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/service/src/com/android/car/systeminterface/WakeLockInterface.java b/service/src/com/android/car/systeminterface/WakeLockInterface.java
index cde6e5b..771d625 100644
--- a/service/src/com/android/car/systeminterface/WakeLockInterface.java
+++ b/service/src/com/android/car/systeminterface/WakeLockInterface.java
@@ -36,7 +36,7 @@ public interface WakeLockInterface {
         DefaultImpl(Context context) {
             PowerManager powerManager =
                     (PowerManager) context.getSystemService(Context.POWER_SERVICE);
-            mFullWakeLock = powerManager.newWakeLock(PowerManager.SCREEN_DIM_WAKE_LOCK,
+            mFullWakeLock = powerManager.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK,
                 CarLog.TAG_POWER);
             mPartialWakeLock = powerManager.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK,
                 CarLog.TAG_POWER);
-- 
2.7.4

